File: backend/src/main/java/org/jhsoft/jhmes/domain/Product.java
// src/main/java/org/jhsoft/jhmes/domain/Product.java
package org.jhsoft.jhmes.domain;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class Product {
    private String productId;
    private String productName;
    private String unit;
    private String description;
    private Boolean isActive;
    private String createdBy;
    private LocalDateTime createdAt;
    private String updatedBy;
    private LocalDateTime updatedAt;
}

File: frontend/src/types/product.ts
export interface Product {
  productId: string;
  productName: string;
  description?: string;
  unit?: string;
  unitConversion?: string;
  specification?: string;
  inspectionType?: string;
  isActive?: boolean;
  createdAt?: string; // ISO date
  updatedAt?: string;
}
